File: Problemafelvetes1_java/Book.java
public class Book {
    private String id;
    private String title;
    private String author;
    private int publicationYear;

    public Book(String id, String title, String author, int publicationYear) {
        this.id = id;
        this.title = title;
        this.author = author;
        this.publicationYear = publicationYear;
    }

    public String getId() {
        return this.id;
    }

    public setId(int id) {
        this.id = id;
    }

    @Override
    public String toString() {
        return "Book{" +
               "id='" + this.id + '\'' +
               ", title='" + this.title + '\'' +
               ", author='" + this.author + '\'' +
               ", publicationYear=" + this.publicationYear +
               '}';
    }
}

File: Problemafelvetes1/Classes/Book.py
class Book:
    _numberOfBooks = 20

    def __init__(self, author, title, publication_year, isbn):
        self._author = author
        self._title = title
        self._publication_year = publication_year
        self._isbn = isbn

    def decreaseBookNumber(self):
        self._numberOfBooks -= 1

    def get_author(self):
        return self._author

    def set_author(self, newAuthor):
        self._author = newAuthor

    author = property(get_author, set_author)